libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/milestone/StepMilestoneTest.java

```diff
@@ -28,21 +24,17 @@
   private RouteProgress routeProgress;
 
   @Before
-  public void setup() throws IOException {
+  public void setup() throws Exception {
```

Comment:
In general, we should avoid inline setup 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=65
Could you extract this block of code into private "factory" methods called and use them only in the tests that actually need it/across the rest of the tests?
IMO, it reflects better what's going on in the test and you don't break the [triple A](https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40) "rule".